editor.c

```diff
@@ -95,3 +95,21 @@ int launch_editor(const char *path, struct strbuf *buffer, const char *const *en
 		return error_errno("could not read file '%s'", path);
 	return 0;
 }
+
+int launch_editor(const char *path, struct strbuf *buffer, const char *const *env)
+{
+	return launch_specified_editor(git_editor(), path, buffer, env);
+}
+
+int launch_sequence_editor(const char *path, struct strbuf *buffer,
+			   const char *const *env)
+{
+	const char *editor = getenv("GIT_SEQUENCE_EDITOR");
+
+	if (!editor)
+		git_config_get_string_const("sequence.editor", &editor);
+	if (!editor)
+		return launch_editor(path, buffer, env);
+
+	return launch_specified_editor(editor, path, buffer, env);
+}
```

Comment:
Maybe have a git_sequence_editor() function that finds the right editor for a sequence and then launch_sequence_editor() could just call `launch_specified_editor(git_sequence_editor(), path, buffer, env);`